README.md

```diff
@@ -253,6 +253,7 @@ _CalDAV or CardDAV servers_
 - [DAViCal](https://www.davical.org/) - Server for calendar sharing (CalDAV) that uses a PostgreSQL database as a data store. ([Source Code](https://gitlab.com/davical-project/davical)) `GPL-2.0` `PHP`
 - [Davis](https://github.com/tchapi/davis/) - A simple, dockerizable and fully translatable admin interface for sabre/dav based on Symfony 5 and Bootstrap 4, largely inspired by Baïkal. `MIT` `PHP`
 - [DecSync CC](https://f-droid.org/packages/org.decsync.cc/) - Serverless contacts, calendar synchronization using your own file syncing method i.e Syncthing, Nextcloud etc. ([Source Code](https://github.com/39aldo39/DecSyncCC)) `GPL-3.0` `Kotlin`
+- [Easy!Appointments](https://easyappointments.org/) - Easy!Appointments is a highly customizable web application that allows your customers to book appointments with you via the web. Supports Google Calendar syncing. ([Demo](https://demo.easyappointments.org/), [Source Code](https://github.com/alextselegidis/easyappointments)) `GPL-3.0` `PHP`
```

Comment:
the Demo an main link are duplicated.

Reply:
Hey @SuperSandro2000 . I thought sub-domains would count as a separate URL for this purpose.

Reply:
Oh, you are right. I didn't catch the demo part. Sorry about that.